### ISSUE TITLE
DH_UPGRADE_SETUPTOOLS equivalent for dh_virtualenv

### ISSUE THREAD

**Issue by lamby** [spotify/dh-virtualenv#196]
Seems like there's ``DH_UPGRADE_SETUPTOO`` when using the "buildsystem" style but no equivalent for the ``--with=python-virtualenv`` style. Is this deliberate? (Want a patch?)

**Comment by jhermann** [comment]
`--preinstall "setuptools>=17.1"` (or another minimal version) works well for me, so the question is what a new option brings to the table. Yes, the simple option is a tad easier to use, but then is that worth it?

**Comment by nailor** [comment]
I agree here that there is no need for a specific new option for this.

**Comment by lamby** [comment]
> So just a docs problem?

Ah, yes :)
